### PR TITLE
Created content-card-9-blue.html

### PR DESCRIPTION
**Pull request by Tapo41**

### CODE REVIEW

**Comment by Tapo41** [comment]
Screenshot_20230210_235201

File: Content Cards/src/content-card-9-blue.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+ <article
+        class="absolute top-[23px] left-[26px] rounded-xl  w-[199px] h-[165px] bg-white"
```
Comment (Rajdip019) [inline Content Cards/src/content-card-9-blue.html:2]: Remove all absolute positioning. @]

**Comment by Tapo41** [comment]
@Rajdip019 Done now 
You can recheck.

File: Content Cards/src/content-card-9-blue.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+ <div class="grid h-screen w-screen place-items-center bg-slate-100">
```
Comment (Rajdip019) [inline Content Cards/src/content-card-9-blue.html:1]: Remove the outer div that centres the whole thing and else all looks fine. And so this for all of them.

**Comment by Rajdip019** [comment]
Looks good @Tapo41 .

**Pull request merged**